app/helpers/middleware_server_helper/textual_summary.rb

```diff
@@ -56,4 +69,9 @@ def textual_lives_on
           )
      }
   end
+
+  def not_yet_started?(server)
+    # domain server case when it hasn't been started yet (it has auto-start flag set to false)
+    !server.middleware_server_group.nil? && @record.properties['Server State'] == 'STOPPED'
```

Comment:
Method `not_yet_started` is always invoked with `@record`. You should probably replace the second part of the condition to `server.properties...`?

If the `server.middleware_server_group` is `nil`, the first part of condition will be `false` which will also be a result of the entire function. Does this really mean that it is not started yet?